lms/envs/aws.py

```diff
@@ -696,6 +705,9 @@
 FIELD_OVERRIDE_PROVIDERS += (
```

Comment:
This surprised me, too. We appear to need them both because not all course content is retrieved using the modulestore's overridden `get_item(s)` methods. In particular, the table of contents listing course content accessible to the user is constructed using the modulestore's [bulk operations](https://github.com/edx/edx-platform/blob/master/lms/djangoapps/courseware/module_render.py#L145-L148) facility. In this case, overrides need to be applied when binding retrieved modules to student context, as we've been doing. So, to answer your question: both settings point to the same override provider because the same provider needs to be applied in two different contexts.